hackathon_site/event/jinja2/event/base.html

```diff
@@ -1,20 +1,36 @@
 <!DOCTYPE html>
 <html lang="en">
     <head>
-        {% block head %}
-        <title>{% block title %}Hackathon Site{% endblock %}</title>
-        <link rel="stylesheet" type="text/css" href="{{ static("event/styles/css/styles.css") }}" />
-        
-        <!-- Compiled and minified CSS -->
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0"/>
+
+        <!-- Material icons  -->
+        <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+        <!-- Compiled and minified CSS for Materialize -->
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css" />
 
-        <!-- Compiled and minified JavaScript -->
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
+        {% block head %}
+        <title>{% block title %}Hackathon Site{% endblock %}</title>
+        <link rel="stylesheet" type="text/css" href="{{ static('event/styles/css/styles.css') }}" />
```

Comment:
Add an empty `head_extends` block below this (but inside the head block) for templates to add css and js files as necessary.